cmlutils/project_entrypoint.py

```diff
@@ -245,3 +246,60 @@ def project_import_cmd(project_name):
         if pimport:
             pimport.terminate_ssh_session()
         exit()
+
+
+@click.group(name="helpers")
+def project_helpers_cmd():
+    """
+    Sub-entrypoint for helpers command
+    """
+
+
+@project_helpers_cmd.command("populate_runtimes_v2")
+def populate_runtimes_v2():
+    project_name = ""
+    config = _read_config_file(
+        os.path.expanduser("~") + "/.cmlutils/import-config.ini", project_name
+    )
+
+    username = config[USERNAME_KEY]
+    url = config[URL_KEY]
+    apiv1_key = config[API_V1_KEY]
+    local_directory = config[OUTPUT_DIR_KEY]
+    ca_path = config[CA_PATH_KEY]
+
+    local_directory = get_absolute_path(local_directory)
+    ca_path = get_absolute_path(ca_path)
+
```

Comment:
Can you configure logging here and logs at various stages? 
_configure_project_command_logging(log_filedir, project_name)